src/content/blog/whats-new-october.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: "What's new in Astro - October"
+description: "Another busy month in Astro! From Hacktoberfest to releasing Page Partials. Read up on everything that happened this month in the Astro universe."
+publishDate: "October 31, 2023"
+authors:
+  - elian
+coverImage: "/src/content/blog/_images/whats-new-october/head.webp"
+socialImage: "/src/content/blog/_images/whats-new-october/og.webp"
+lang: "en"
+---
+
+## Astro releases
+
+Astro's development keeps marching forward, with several notable releases in October:
+
+- Astro 3.2: This update gives you more control over the browser history stack, enables page navigation with View Transitions, includes a route announcer for screen readers, and integrations can now dynamically add other integrations. [Learn more](/blog/astro-320/).
+- Astro 3.3: This release introduces an experimental `<Picture />` component, enhances syntax highlighting, brings package provenance, and adds various quality-of-life improvements. [Read more](/blog/astro-330/).
+- Astro 3.4: Introduction Page Partials, along with improvements in image optimization and a first preview of an experimental dev overlay. [Discover more](/blog/astro-340/).
+
+See the `#announcements` channel in Discord for all releases across the Astro repos, including Starlight, the Astro VS Code extension, compiler, packages and integrations! Follow [Astro on GitHub](https://github.com/withastro/) for all updates.
+
+## Hacktoberfest
+
+[Hacktober](https://hacktoberfest.com/) is officially over now! Thank you all for your amazing contributions and PRs! Here are some of our statistics:
+
+- 2 Astro core PRs
+- 49 Starlight PRs
+- 62 Docs PRs
+
+## Docs
+
+We extended the tutorial with some requested features!
+
+- [Content Collections](https://docs.astro.build/en/tutorials/add-content-collections/)
+- [View Transitions](https://docs.astro.build/en/tutorials/add-view-transitions/)
+
+"Talking and Doc'ing" is our weekly live Discord call where we involve the community as we work on docs projects! Often we jump in a shared CodeSpace or HackMD document and everyone can participate. This months's activities included:
```

Comment:
A few nits here!
- I believe the correct word here would be "into" rather than "in"
- By "CodeSpace" do you mean the GitHub product? Then it would be "Codespace" instead, so that our spelling is 1:1 with how it's written in their official copy!
- The spelling here would be "month's" rather than "months's"
```suggestion
"Talking and Doc'ing" is our weekly live Discord call where we involve the community as we work on docs projects! Often we jump into a shared Codespace or HackMD document and everyone can participate. This month's activities included:
```